### PR TITLE
Update: bolded headers to h3, h4, h5

### PR DESCRIPTION
**Pull request by MimiDumpling**
Updated bolded headers for consistency with h3, h4, and h5. H5s used sparingly.

### CODE REVIEW

File: src/docs/product/accounts/pricing.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ If your projects have a high volume of events, you can control how many errors S
 
 ## FAQ
 
-**Is there detailed information for each plan?**
+#### Is there detailed information for each plan?
```
Comment (philipphofmann) [inline src/docs/product/accounts/pricing.mdx:57]: Shouldn't this be `###` if FAQ has `##`?

File: src/docs/product/releases/health.mdx [hunk 2]
```diff
@@ -102,25 +102,31 @@ Data filters are only partially supported for session events, and they might be
 
 ### Equivalent Filters
 
-**release filter**
-This filter discards events of a particular release. If sessions are started or updated and would match this filter, sessions are silently discarded entirely. This is consistent with error events.
+Release Filter
```
Comment (philipphofmann) [inline src/docs/product/releases/health.mdx:105]: The changes in this file make it way cleaner. I like it a lot 👏

**Review by philipphofmann — APPROVED** [review]
One small comment from my side, but I like the changes. Makes the docs a bit better IMO 👏

File: src/docs/product/accounts/pricing.mdx (hunk 1 above)
Comment (PeloWriter) [reply src/docs/product/accounts/pricing.mdx:57]: Structurally, yes. But in this case, the change was made to fix a bolding issue, so an H4 is fine. Thanks for reviewing.

**Pull request merged**